_data/hosting.yml

```diff
@@ -436,8 +436,7 @@ websites:
       tfa: Yes
       sms: Yes
```

Comment:
Because 2FA requires setting up a software method and SMS is only used as a backup method, I'd categorize this site as not supporting 2FA.
```suggestion
```